mongo/options/aggregateoptions.go

```diff
@@ -50,6 +50,10 @@ type AggregateOptions struct {
 	// Values must be constant or closed expressions that do not reference document fields. Parameters can then be
 	// accessed as variables in an aggregate expression context (e.g. "$$var").
 	Let interface{}
+
+	// Custom options to be added to aggregate expression. Key-value pairs should correlate with desired option names
+	// and values. Values must be Marshalable.
```

Comment:
Suggest noting that non-custom options (or perhaps "named options") should be preferred when possible. Perhaps this:

```suggestion
	// and values. Values must be Marshalable. Custom options may conflict with non-custom options.
	// Custom options bypass client-side validation. Prefer using non-custom options when possible.
```

Reply:
Good idea, added.